### DISCUSSION TITLE
Tempate Render Error messaging improvement: line numbers do not account for front matter

### DISCUSSION THREAD

**Discussion by zachleat** [11ty/eleventy#4012] [Enhancement Queue]
![image](https://user-images.githubusercontent.com/39355/49449587-d37fdb80-f7a0-11e8-9fc5-5eadf5cd64b5.png)

For example, the error was on file line 92. Front matter was 11 lines long. Line reported in the error message was Line 81.

**Comment by zellwk** [comment]
Oh yep I noticed this too, but it wasn't too big of an issue :D

**Comment by zachleat** [comment]
This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!
